build.gradle

```diff
@@ -173,11 +174,13 @@ dependencies {
   implementation group: 'org.springframework.boot', name: 'spring-boot-starter-validation'
   implementation group: 'com.github.hmcts', name: 'pip-data-models', version: '1.0'
   implementation 'org.postgresql:postgresql'
+  implementation 'org.springframework.boot:spring-boot-starter-webflux'
```

Comment:
dont forget to add versions to dependencies, were gunna end up to deep in a gradle file with random versions, also keep them full gradle implementations rather than shortened ones

Reply:
Honestly, I had no idea there was best practice for this. Thanks a lot for this comment, it's sent me down the right path here. I've gone through all the dependencies and bumped versions to most recent (stable) releases etc and everything seems to build okay.

Reply:
nice work!